### ISSUE TITLE
Missing onground movement

### ISSUE THREAD

**Issue by renanbuosiferreira** [xoolive/traffic#108]
Hi!

I'm comparing data collected from a friend's ADS-B antenna, which are shared with opensky, and the data that I get with the API, and points on surface are missing, as we can see in the attached figures.

![image(1)](https://user-images.githubusercontent.com/60225043/99260247-e4914e00-27f9-11eb-9c74-739fc407407d.png)
![image](https://user-images.githubusercontent.com/60225043/99260250-e529e480-27f9-11eb-8f1b-8bc225f8e059.png)

Is Opensky or the library filtering it out? How can I get all data?

**Comment by xoolive** [comment]
Could you please share the information (timestamp and (icao24 or callsign)) about the flight you are plotting? I will have a look at data stored in base.

**Comment by renanbuosiferreira** [comment]
Reference data:
time                                       2020-09-28 09:13:48
icao24                                                  3c7062
lat                                                    -22.957
lon                                                   -47.2049
velocity                                               96.7374
heading                                                128.089
vertrate                                              -5.85216
callsign                                               GEC8264
onground                                                 False
alert                                                    False
spi                                                      False
squawk                                                     NaN
baroaltitude                                           1059.18
geoaltitude                                            1089.66
lastposupdate                                      1.60128e+09
lastcontact                                        1.60128e+09
hour                                                1601283600
geometry         POINT (-47.20494588216144 -22.95695676641952)

**Comment by xoolive** [comment]
Thank you for your help.

So look, I first got the data from the state vectors:
```python
from traffic.data import opensky

f = opensky.history(
    "2020-09-28 09:05", "2020-09-28 09:30",
    icao24="3c7062", return_flight=True
)
```

and here, I see there are signals until quite late, but the last positional information are timestamped at 09:15.

<img width="705" alt="Capture d’écran 2020-11-16 à 15 33 26" src="https://user-images.githubusercontent.com/1360812/99264681-14a01780-2821-11eb-83da-fd04d20ab6d7.png">

Just to confirm about data availability, I got the raw data and decoded it again:
```python
d = opensky.rawdata(f.start, f.stop, icao24=f.icao24)
t = d.decode("SBKP")
t[f.icao24].map_leaflet(zoom=13)
```

and...

<img width="717" alt="Capture d’écran 2020-11-16 à 15 43 22" src="https://user-images.githubusercontent.com/1360812/99265868-78771000-2822-11eb-9294-12503ecbf093.png">

The decoder in `traffic` is based on https://github.com/junzis/pyModeS but the decoder in OpenSky is a Java implementation https://github.com/openskynetwork/java-adsb maybe there is another issue on the backend then that I am not aware of.
